addons/storyshots/storyshots-core/README.md

```diff
@@ -549,8 +554,27 @@ initStoryshots({
 ```
 
 This option needs to be set if either:
-* the multiSnapshot function is used to create multiple snapshot files (i.e. one per story), since it ignores any serializers specified in your jest config.
-* serializers not specified in your jest config should be used when snapshotting stories.
+
+- the multiSnapshot function is used to create multiple snapshot files (i.e. one per story), since it ignores any serializers specified in your jest config.
+- serializers not specified in your jest config should be used when snapshotting stories.
+
+### `storyParam`
```

Comment:
I need to update the docs now that the main storyshots docs include the `disable` parameter, and possibly undo my editor's excited and sometimes ridiculous markdown 

- [ ] update docs to reflect addition of `disable` documentation
- [ ] remove auto-formatting changes